cluster/gce/gci/configure.sh

```diff
@@ -33,6 +33,13 @@ DEFAULT_CRICTL_SHA1="5c18f4e52ab524d429063b78d086dd18b894aae7"
 DEFAULT_MOUNTER_TAR_SHA="8003b798cf33c7f91320cd6ee5cec4fa22244571"
 ###
 
+# Ability to inject custom versions (Ubuntu OS images ONLY)
+# if KUBE_CONTAINERD_VERSION_OVERRIDE or KUBE_RUNC_VERSION_OVERRIDE
+# is set to empty then we do not override the version(s) and just
+# use whatever is in the default installation of containerd package
```

Comment:
I would have expected this to be flipped... by default, use the installed versions, and require overrides to actively pull/install packages during CI

Reply:
also, if this install process is ubuntu only, and triggers install, maybe name the envvars to match that, e.g. `UBUNTU_INSTALL_CONTAINERD_VERSION` and `UBUNTU_INSTALL_RUNC_VERSION`?

Reply:
i wanted the new CI job to test the latest from containerd (1.3.3) and runc (v10) by default, these are not available currently. 

yep, we can change it around once i get the CI jobs go green (and tweak what we want be defaults and env vars)

Reply:
making the change now. just going to use `UBUNTU_INSTALL_CONTAINERD_VERSION` and `UBUNTU_INSTALL_RUNC_VERSION` everywhere